.pre-commit-config.yaml

```diff
@@ -17,6 +17,11 @@ repos:
     hooks:
       - id: end-of-file-fixer
       - id: trailing-whitespace
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
+        args: ["--strict"]
```

Comment:
This will be a bit disruptive to local development.. running `pre-commit run --all-files` locally (which I do many times during local development)... will produce all of of output and fail until we've fixed all the issues.

Would you consider the following instead?

* add all the rules that currently produce errors to `.yamllint.yml` in this PR with `:disable`
* remove the `SKIP=yamllint` in `lint-python-bash.sh`
* enable more and more rules over each of the next set of PRs.

That would prevent any *new* issues from being introduced on `master` while allowing `pre-commit run --all-files` to continue working locally.

Reply:
Makes sense! Thank you very much for the suggestion! I just pushed 4adf207fdaed541a9d444c399bd06732bdea17a7 with these changes.